packages/visual-group/src/variable/simple-var.js

```diff
@@ -119,4 +119,16 @@ export default class SimpleVariable extends Variable {
     equals (varInst) {
         return this.oneVar() === varInst.oneVar();
     }
+
+    /**
+     * Returns the display name of the field, in case of composite-field it returns displayName for first field.
+     *
+     * @return {string} return the display name.
+     */
+    displayName () {
```

Comment:
What happens when a composite variable is passed?

Reply:
It considers only the first variable like the `oneVar()` method.